deploy/kubernetes/base/controller.yaml

```diff
@@ -0,0 +1,77 @@
+---
+# Controller Service
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: efs-csi-controller
+  namespace: kube-system
+  labels:
+    app.kubernetes.io/name: aws-efs-csi-driver
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: efs-csi-controller
+      app.kubernetes.io/name: aws-efs-csi-driver
+  template:
+    metadata:
+      labels:
+        app: efs-csi-controller
+        app.kubernetes.io/name: aws-efs-csi-driver
+    spec:
+      nodeSelector:
+        kubernetes.io/os: linux
+      serviceAccountName: efs-csi-controller-sa
+      priorityClassName: system-cluster-critical
+      tolerations:
+        - operator: Exists
```

Comment:
I copied it from an upstream template, I'm not too sure of this one.

Reply:
"An empty key with operator Exists matches all keys, values and effects which means this will tolerate everything."

https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/

Reply:
i think ther has been discussion in the past that toleration all taints causes issues for users trying to drain nodes and such expecting NoSchedule to actually mean NoSchedule but let's just copy the template